ci/scripts/github/common.sh

```diff
@@ -37,9 +43,10 @@ export CMAKE_BUILD_WITH_CODECOV="-DCMAKE_BUILD_TYPE=Debug -DSRF_ENABLE_CODECOV=O
 export GIT_DEPTH=1000
 
 # For PRs, $GIT_BRANCH is like: pull-request/989
-REPO_NAME=$(basename "${GIT_URL}" .git)
-ORG_NAME=$(basename "$(dirname "${GIT_URL}")")
-PR_NUM="${GIT_BRANCH##*/}"
+REPO_NAME=$(basename "${GITHUB_REPOSITORY}")
+ORG_NAME="${GITHUB_REPOSITORY_OWNER}"
+PR_NUM="${GITHUB_REF_NAME##*/}"
```

Comment:
Will this fail if we arent in a PR? i.e. on a branch build?

Reply:
I tested this, on a branch build `PR_NUM` will just be the branch name.
```
(base) root@roci:/SRF# export GITHUB_REF_NAME=branch-22.11
(base) root@roci:/SRF# PR_NUM="${GITHUB_REF_NAME##*/}"
(base) root@roci:/SRF# echo $?
0
(base) root@roci:/SRF# echo $PR_NUM
branch-22.11
```

The `PR_NUM` var is used in two places:

1. As part of the S3 artifact url:
`export ARTIFACT_ENDPOINT="/pull-request/${PR_NUM}/${GIT_COMMIT}/${NVARCH}/${BUILD_CC}"`

1. As part of the check script where we use the github rest api to fetch the changes. 


The check script is skipped on branch builds and I think it works ok as part of the s3 url.

Reply:
Sounds good